2024-03-01 12:11:24 ERROR Elements not present after 0.5 seconds on page: https://capital.com/trade-ethereum-classic
2024-03-01 12:11:24 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 84, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 302, in element_is_visible
    return Wait(self.driver, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcompyse-CcJ6hTD8-py3.12/lib/python3.12/site-packages/selenium/webdriver/support/wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
#0 0x5563e0323e93 <unknown>
#1 0x5563e001bce6 <unknown>
#2 0x5563e0066e48 <unknown>
#3 0x5563e0066f01 <unknown>
#4 0x5563e00aa3f4 <unknown>
#5 0x5563e0088edd <unknown>
#6 0x5563e00a7899 <unknown>
#7 0x5563e0088c53 <unknown>
#8 0x5563e0059db3 <unknown>
#9 0x5563e005a77e <unknown>
#10 0x5563e02e97cb <unknown>
#11 0x5563e02ed7e5 <unknown>
#12 0x5563e02d70e1 <unknown>
#13 0x5563e02ee372 <unknown>
#14 0x5563e02bb1bf <unknown>
#15 0x5563e0312488 <unknown>
#16 0x5563e0312683 <unknown>
#17 0x5563e0323044 <unknown>
#18 0x7fdbb2094ac3 <unknown>

2024-03-01 12:11:27 ERROR Elements not present after 0.5 seconds on page: https://capital.com/trade-ethereum-classic
2024-03-01 12:11:27 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 84, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 302, in element_is_visible
    return Wait(self.driver, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcompyse-CcJ6hTD8-py3.12/lib/python3.12/site-packages/selenium/webdriver/support/wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
#0 0x5563e0323e93 <unknown>
#1 0x5563e001bce6 <unknown>
#2 0x5563e0066e48 <unknown>
#3 0x5563e0066f01 <unknown>
#4 0x5563e00aa3f4 <unknown>
#5 0x5563e0088edd <unknown>
#6 0x5563e00a7899 <unknown>
#7 0x5563e0088c53 <unknown>
#8 0x5563e0059db3 <unknown>
#9 0x5563e005a77e <unknown>
#10 0x5563e02e97cb <unknown>
#11 0x5563e02ed7e5 <unknown>
#12 0x5563e02d70e1 <unknown>
#13 0x5563e02ee372 <unknown>
#14 0x5563e02bb1bf <unknown>
#15 0x5563e0312488 <unknown>
#16 0x5563e0312683 <unknown>
#17 0x5563e0323044 <unknown>
#18 0x7fdbb2094ac3 <unknown>

2024-03-01 12:11:30 ERROR Elements not present after 0.5 seconds on page: https://capital.com/trade-ethereum-classic
2024-03-01 12:11:30 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 84, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 302, in element_is_visible
    return Wait(self.driver, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcompyse-CcJ6hTD8-py3.12/lib/python3.12/site-packages/selenium/webdriver/support/wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
#0 0x5563e0323e93 <unknown>
#1 0x5563e001bce6 <unknown>
#2 0x5563e0066e48 <unknown>
#3 0x5563e0066f01 <unknown>
#4 0x5563e00aa3f4 <unknown>
#5 0x5563e0088edd <unknown>
#6 0x5563e00a7899 <unknown>
#7 0x5563e0088c53 <unknown>
#8 0x5563e0059db3 <unknown>
#9 0x5563e005a77e <unknown>
#10 0x5563e02e97cb <unknown>
#11 0x5563e02ed7e5 <unknown>
#12 0x5563e02d70e1 <unknown>
#13 0x5563e02ee372 <unknown>
#14 0x5563e02bb1bf <unknown>
#15 0x5563e0312488 <unknown>
#16 0x5563e0312683 <unknown>
#17 0x5563e0323044 <unknown>
#18 0x7fdbb2094ac3 <unknown>

2024-03-01 12:11:33 ERROR Elements not present after 0.5 seconds on page: https://capital.com/trade-ethereum-classic
2024-03-01 12:11:33 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 84, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 302, in element_is_visible
    return Wait(self.driver, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcompyse-CcJ6hTD8-py3.12/lib/python3.12/site-packages/selenium/webdriver/support/wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
#0 0x5563e0323e93 <unknown>
#1 0x5563e001bce6 <unknown>
#2 0x5563e0066e48 <unknown>
#3 0x5563e0066f01 <unknown>
#4 0x5563e00aa3f4 <unknown>
#5 0x5563e0088edd <unknown>
#6 0x5563e00a7899 <unknown>
#7 0x5563e0088c53 <unknown>
#8 0x5563e0059db3 <unknown>
#9 0x5563e005a77e <unknown>
#10 0x5563e02e97cb <unknown>
#11 0x5563e02ed7e5 <unknown>
#12 0x5563e02d70e1 <unknown>
#13 0x5563e02ee372 <unknown>
#14 0x5563e02bb1bf <unknown>
#15 0x5563e0312488 <unknown>
#16 0x5563e0312683 <unknown>
#17 0x5563e0323044 <unknown>
#18 0x7fdbb2094ac3 <unknown>